versions/3_12/samples/terminal/process_payment.py

```diff
@@ -11,13 +11,17 @@
 
 def send_terminal_payment() -> None:
     show_tips = bool(input("\n\nDo you want to enable Terminal to show Tips (True/False):"))
+    payment_method = kody_model.PaymentMethod(
+        payment_method_type=input("\n\nChoose payment method type (ALIPAY, WECHAT or CARD (default):")
```

Comment:
When you suggest 'default' does that mean I can just hit 'enter' and it chooses that default?

Reply:
I see the predicament. Not we do not pass a default value in case of blank enter.